packages/atomic/src/components.d.ts

```diff
@@ -16,6 +16,11 @@ import { InitializationOptions } from "./components/atomic-search-interface/atom
 import { StandaloneSearchBoxData } from "./utils/local-storage-utils";
 export namespace Components {
     interface AtomicAriaLive {
+        /**
+          * Only the `atomic-aria-live` element with the greatest priority will be used to announce changes in the search interface.
```

Comment:
I tried to add `@internal` here but it didn't stick for some reason?

The reason I went for this solution is because I'm afraid that developers using JS frameworks may dynamically add or remove `aria-live`, and I wanted a solution that works as consistently as possible in those cases.

Reply:
That's generated code, from the component itself.

And even if you add things to the definition file, that would be wrong since it would not match the implementation.